oelint_adv/cls_rule.py

```diff
@@ -105,13 +105,11 @@ def finding(self, _file, _line, override_msg=None, appendix=None):
         if get_relpaths():
             _path = os.path.relpath(_path, os.getcwd())
 
-        _style = ''
         _color = ''
         if get_colorize():
             _color = get_color_by_severity(_severity)
-            _style = Style.RESET_ALL
 
-        return [(_line, f'{_color}{_path}:{_line}:{_severity}:{_display_id}{override_msg}{_style}')]
+        return [(_line, f'{_color}{_path}:{_line}:{_severity}{_display_id}: {override_msg}{Style.RESET_ALL}')]
```

Comment:
I get your point, but as I said in another PR - I simply don't want to change the way the output is shown with default settings.
What I could offer is to make the complete output message configurable. This would give users unlimited possibilities to do their own

Reply:
Would that be okay for you @HerrMuellerluedenscheid?

Reply:
Looks good 👍